### PR TITLE
Add documentation for working with Web Components

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

This is aimed at documenting the admittedly awkward process of working on the parallel `web-components` branch.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by k80bowman — APPROVED** [review]
This will be really helpful, thank you for adding it!

File: README.md [hunk 1]
```diff
@@ -22,6 +22,17 @@ From the project root, run the following commands:
...
 
+## Web Components
+
+The Design System team is working on replacing the React components in this library with Web Component versions. This isn't _just_ a React replacement however - part of the goal is to have true components that can be used outside of React in plain HTML pages.
+
+For now these Web Components live on the [`web-components` branch](https://github.com/department-of-veterans-affairs/component-library/tree/web-components). In order to add a new component or make changes to an existing one, you must:
+
+1. Create a new branch off of `web-components`
+1. Have the PR be based off of `web-components` (not `master`)
+1. Once the PR is merged, create a Github Release with `web-components` as the base, where the version is `wc-vX.Y.Z`. with `X.Y.Z` being the semantic version.
```
Comment (k80bowman) [inline README.md:33]: Are there any binaries that need to be added to the release? Might want to add details on things like tagging, titling, adding info, etc. Screenshots might be helpful.

Comment (bkjohnson) [reply README.md:33]: The binaries happen automatically. I've been following [this](https://keepachangelog.com/en/1.0.0/) for the description section, so I can add that in.

File: README.md [hunk 2]
```diff
@@ -22,6 +22,17 @@ From the project root, run the following commands:
...
 1. `yarn storybook`
 
+## Web Components
+
+The Design System team is working on replacing the React components in this library with Web Component versions. This isn't _just_ a React replacement however - part of the goal is to have true components that can be used outside of React in plain HTML pages.
+
+For now these Web Components live on the [`web-components` branch](https://github.com/department-of-veterans-affairs/component-library/tree/web-components). In order to add a new component or make changes to an existing one, you must:
+
+1. Create a new branch off of `web-components`
+1. Have the PR be based off of `web-components` (not `master`)
```
Comment (cvalarida) [inline README.md:32]: Bonus points for a link or screenshot explaining what this means. I can see somebody getting confused. ("Well yes, I branched off `web-components`, so the PR is based on that branch...?" But this is specifically talking about the _merge_ base.)

**Review by cvalarida — APPROVED** [review]

**Pull request merged**